_posts/2022-06-03-gaps-in-bitcoin-dev-funding.markdown

```diff
@@ -0,0 +1,30 @@
+
+Bridge grants are short-term grants, somewhere between 3 and 6 months, to wean a prospect off their job and give them a runway to show what they can do. They won't get rich from this funding, but it will be enough to avoid starvation and should inspire them to spend their time wisely establishing a track record that will attract a full-fledged grant.
+
+## A manager
+
+No strings attached funding provides the freedom to do meaningful work. In an industrialist society that no longer offers much respect to the individual worker, it reflects the trust of the grantee to choose the work that needs to be done. It's a thing of beauty.
+
+But, without strings, grantees don't go through the same growth of learning how to be professional -- AKA, how to show up when they don't feel like it. The kind of pain that comes with showing up to a bad job day after day because you made a promise to be there. No strings attached open-source funding doesn't help develop young people to be critical of management or understand when they need to quit a job. When a manager isn't parceling out tasks or providing (even poorly done) performance reviews, I believe it is harder for a green dev to get the feedback they need to grow as a developer and generally be consistently productive. Because FOSS doesn't signal whether you are performing well, I've seen people become demotivated and discouraged without a feedback structure that works (see wildly successful talent recruitment and retention of large tech companies).
+
+## A career
+
+Bitcoin open-source development isn't yet a career. Even in the US, where taking the road less traveled is culturally celebrated, health care is tied to employment. There is motivational value in career progression. While I don't believe in titles (see current descriptive title - Head of Special Projects), demonstrating career progression is helpful to mitigate the criticisms of friends and family who are already skeptical of the open-source path.
+
+## Something else
+
+We need a prospect league that is a scaled-down version of the current grant system with a greater margin of error and more structure. This development program provides the financial safety and modeling to invite more of those that otherwise couldn't make the jump to bitcoin open-source development. This is the strategy behind programs like Summer of Bitcoin and Qala. Get a foothold. Find promising people to invest more in. We have to do it on a larger scale. FOSS development [can be hard](/bitcoin/managing-yourself-in-open-source/) and so we can either say that devs without 5+ years of wading through industry jobs are the only people that should attempt it, or we can create better scaffolding and structure for those that are greener.
```

Comment:
```suggestion
We need a prospect league that is a scaled-down version of the current grant system with a greater margin of error and more structure. This development program provides the financial safety and modeling to invite more of those that otherwise couldn't make the jump to bitcoin open-source development. This is the strategy behind programs like Summer of Bitcoin and Qala. Get a foothold. Find promising people to invest more in. We have to do it on a larger scale. FOSS development [can be hard](/bitcoin/managing-yourself-in-open-source/). We can either say that devs without 5+ years of wading through industry jobs are the only people who should attempt it, or we can create better scaffolding and structure for those that are greener.
```